qa/suites/fs/functional/tasks/snap-schedule.yaml

```diff
@@ -15,6 +15,7 @@ overrides:
       - is full \(reached quota
       - POOL_FULL
       - POOL_BACKFILLFULL
+      - pg peering \(PG_AVAILABILITY\)
```

Comment:
So the failure is annoying but it represents a real bug we should not squelch. Just document that it's a known problem when it shows up in qa results.

Let's remove this ignorelist addition.

Reply:
okay, I'll remove it from the ignorelist